notebooks/7-rotation-speeds/interm_rotation_rate.ipynb

```diff
@@ -0,0 +1,2663 @@
+{
```

Comment:
<blockquote>The rotation rate of a stars tel us about their evolution</blockquote><p>Typo, "tell"</p><p> </p><p>Also I think we now know more about where it's going and can be more specific at the end - e.g. "How rotation rates vary with flare rate across the M dwarf population." (And maybe a line above about the physics of the possible connection? What is a flare and why might they be related to rotation rate?)</p><p> </p><p>Include linked table of contents? (I think the template has an example?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>